### PR TITLE
Added CRUD tests for F-130 and F-131 for createCase and createEvent …

### PR DESCRIPTION
**Pull request by radhikahollakv**
…endpoints
createCase /case-types/{caseTypeId}
createEvent /cases/{caseId}/events

### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/RDM-12531

### Change description ###

Added the following Scenarios:
Scenario: User cannot submit case creation without CaseType C Access
Scenario: User cannot submit case creation without CaseEvent C Access
Scenario: User cannot create an event with missing CaseEvent C Access
Scenario: User successfully updates a case with a field with no CaseField Access R does not see the field in the response

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x ] No
```

### CODE REVIEW

File: src/aat/resources/features/F-131 - AC CRUD createEvent external/S-131.1.td.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  ],
+
+  "request": {
+    "_extends_": "Common_Request",
+    "pathVariables": {
+      "caseId": "${[scenarioContext][childContexts][F-131_CreateCase][testData][actualResponse][body][id]}"
+    },
+
+    "headers": {
+      "experimental": true
```
Comment (noronhaa) [inline src/aat/resources/features/F-131 - AC CRUD createEvent external/S-131.1.td.json:17]: we can remove this line as it's in the base file

File: src/aat/resources/features/F-131 - AC CRUD createEvent external/S-131.1.td.json [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    },
+    "body": {
+      "data": {
+        "TextField4" : "value4"
+      },
+      "event": {
+        "id" : "event1"
+      },
+      "ignore_warning": false,
```
Comment (noronhaa) [inline src/aat/resources/features/F-131 - AC CRUD createEvent external/S-131.1.td.json:26]: we can remove this line as it's in the base file

File: src/aat/resources/features/F-131 - AC CRUD createEvent external/S-131.4.td.json [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+  ],
+
+  "request": {
+    "_extends_": "Common_Request",
+    "pathVariables": {
+      "caseId": "${[scenarioContext][childContexts][F-131_CreateCase][testData][actualResponse][body][id]}"
+    },
+
+    "headers": {
+      "experimental": true
```
Comment (noronhaa) [inline src/aat/resources/features/F-131 - AC CRUD createEvent external/S-131.4.td.json:18]: we can remove this line as it's in the base file

File: src/aat/resources/features/F-131 - AC CRUD createEvent external/S-131.4.td.json [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+    },
+    "body": {
+      "data": {
+        "TextField2" : "value2"
+      },
+      "event": {
+        "id" : "event3"
+      },
+      "ignore_warning": false,
```
Comment (noronhaa) [inline src/aat/resources/features/F-131 - AC CRUD createEvent external/S-131.4.td.json:27]: we can remove this line as it's in the base file

File: src/aat/resources/features/F-131 - AC CRUD createEvent external/S-131.5.td.json [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+  ],
+
+  "request": {
+    "_extends_": "Common_Request",
+    "pathVariables": {
+      "caseId": "${[scenarioContext][childContexts][F-131.5_CreateCase][testData][actualResponse][body][id]}"
+    },
+
+    "headers": {
+      "experimental": true
```
Comment (noronhaa) [inline src/aat/resources/features/F-131 - AC CRUD createEvent external/S-131.5.td.json:17]: we can remove this line as it's in the base file

File: src/aat/resources/features/F-131 - AC CRUD createEvent external/S-131.5.td.json [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+    },
+    "body": {
+      "data": {
+        "TextField" : "value"
+      },
+      "event": {
+        "id" : "event1"
+      },
+      "ignore_warning": false,
+      "event_token": "${[scenarioContext][childContexts][S-131.5_Token_Creation][testData][actualResponse][body][token]}"
```
Comment (noronhaa) [inline src/aat/resources/features/F-131 - AC CRUD createEvent external/S-131.5.td.json:27]: we can remove this line as it's in the base file

File: src/aat/resources/features/F-131 - AC CRUD createEvent external/S-131.5.td.json [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+    }
+  },
+
+  "expectedResponse": {
+    "_extends_" : "Common_404_Response",
+      "headers": {
```
Comment (noronhaa) [inline src/aat/resources/features/F-131 - AC CRUD createEvent external/S-131.5.td.json:33]: I think most if not all these headers can be removes as they will be in base files

File: src/aat/resources/features/F-131 - AC CRUD createEvent external/S-131.4.td.json [hunk 8]
```diff
@@ -0,0 +1,74 @@
...
+      "event_token": "${[scenarioContext][childContexts][S-131.4_Token_Creation][testData][actualResponse][body][token]}"
+    }
+  },
+
+  "expectedResponse": {
+    "_extends_" : "Common_201_Response",
+    "headers": {
```
Comment (noronhaa) [inline src/aat/resources/features/F-131 - AC CRUD createEvent external/S-131.4.td.json:34]: most of these headers can be removes as they are in base files

**Review by noronhaa — CHANGES_REQUESTED** [review]

File: src/aat/resources/features/F-131 - AC CRUD createEvent external/S-131.1.td.json (hunk 2 above)
Comment (radhikahollakv) [reply src/aat/resources/features/F-131 - AC CRUD createEvent external/S-131.1.td.json]: Have removed this line now

File: src/aat/resources/features/F-131 - AC CRUD createEvent external/S-131.1.td.json (hunk 1 above)
Comment (radhikahollakv) [reply src/aat/resources/features/F-131 - AC CRUD createEvent external/S-131.1.td.json]: Have removed this line now

File: src/aat/resources/features/F-131 - AC CRUD createEvent external/S-131.4.td.json (hunk 3 above)
Comment (radhikahollakv) [reply src/aat/resources/features/F-131 - AC CRUD createEvent external/S-131.4.td.json]: Removed this line as it's in the base file

File: src/aat/resources/features/F-131 - AC CRUD createEvent external/S-131.4.td.json (hunk 4 above)
Comment (radhikahollakv) [reply src/aat/resources/features/F-131 - AC CRUD createEvent external/S-131.4.td.json]: Removed this line as it's in the base file

File: src/aat/resources/features/F-131 - AC CRUD createEvent external/S-131.5.td.json (hunk 5 above)
Comment (radhikahollakv) [reply src/aat/resources/features/F-131 - AC CRUD createEvent external/S-131.5.td.json]: Removed this line as its in the base file

File: src/aat/resources/features/F-131 - AC CRUD createEvent external/S-131.5.td.json (hunk 6 above)
Comment (radhikahollakv) [reply src/aat/resources/features/F-131 - AC CRUD createEvent external/S-131.5.td.json]: Removed this line as its in the base file

File: src/aat/resources/features/F-131 - AC CRUD createEvent external/S-131.5.td.json (hunk 7 above)
Comment (radhikahollakv) [reply src/aat/resources/features/F-131 - AC CRUD createEvent external/S-131.5.td.json]: Yes, have removed the once that are repeated from the base files

File: src/aat/resources/features/F-131 - AC CRUD createEvent external/S-131.4.td.json (hunk 8 above)
Comment (radhikahollakv) [reply src/aat/resources/features/F-131 - AC CRUD createEvent external/S-131.4.td.json]: Removed headers that are in the base files

**Pull request merged**